perun-core/src/test/java/cz/metacentrum/perun/core/impl/UtilsIntegrationTest.java

```diff
@@ -334,6 +338,110 @@ public void validateGroupNameThrowsACorrectExceptionForInvalidRegex() {
 				.isThrownBy(() -> Utils.validateGroupName("members", invalidPattern));
 	}
 
+	@Test
+	public void checkHostDestination() throws Exception {
+		System.out.println("Utils.checkHostDestination");
+
+		Destination destination = new Destination();
+		destination.setType(Destination.DESTINATIONHOSTTYPE);
+
+		destination.setDestination("192.168.1.1");
+		Utils.checkDestination(destination);
+
+		destination.setDestination("hostname.test");
+		Utils.checkDestination(destination);
+	}
+
+	@Test
+	public void checkMailDestination() throws Exception {
+		System.out.println("Utils.checkHostDestination");
```

Comment:
This should be the same as each test method name, but is same as first test in the most of these tests.

Reply:
Fixed.